main.go

```diff
@@ -24,6 +25,13 @@ var (
 	bundleFieldsMap    = warehouse.BundleFields()
 )
 
+const (
+	maxAttempts int = 3
```

Comment:
Just a NIT: Hauser makes different kinds of API calls, to BigQuery, RedShift, FullStory etc. I think it would help to be clear to say when exactly these constants will be used. 

So, maybe `maxRetryAttempts` would be more apt along with a comment that says where it will be used (to make fullstory fetch calls).